doc/manual/tuto.md

```diff
@@ -186,3 +186,66 @@ be expected as we removed it earlier.
 The second part shows the contract attribute that produced the error.
 In this case it's showing that `name` should be a `Str`, and as there
 is no `optional` keyword, this attribute must be set.
+
+## Step 7: Document your Nickel file
+
+Nickel supports metadata on fields, this mean you can actually write
+documentation to explain your Nickel code. This way, your team can
+easily reuse a Nickel code without having to ask the author about a
+contract usage.
```

Comment:
Suggestion of wording, to make this paragraph a tad lighter. Feel free to do what you want with it, including ignoring it :slightly_smiling_face: 

```suggestion
Beside contracts, you can attach various kinds of metadata to
record fields in Nickel. In particular, it is good practice to
include documentation when writing contracts.
```